client/src/main/com/sinch/sdk/auth/adapters/ApplicationAuthManager.java

```diff
@@ -97,4 +99,49 @@ private String encode(String stringToSign) {
       throw new ApiAuthException(e);
     }
   }
+
+  public boolean checkAuthentication(
```

Comment:
I have a problem understanding what method do. Check auth for ... what exactly? It returns bool - so checkAuth == true what it means?
Something like `isValidAuth` or `ValidateRequest` or `ValidateAuth`

Reply:
+1: Naming could have been better: refactored to `validateAuthenticatedRequest`
-1: About doc: this is an internals so not a lot of doc ... if function name is judiciously selects and your comment highlighted it was not -> thank you
From SDK users's usage perspective doc is present (but changed the name accordingly): https://github.com/sinch/sinch-sdk-java/blob/main/client/src/main/com/sinch/sdk/domains/verification/WebHooksService.java#L38